core/client/src/test/java/alluxio/client/RemoteBlockWriterTest.java

```diff
@@ -0,0 +1,37 @@
+package alluxio.client;
```

Comment:
@gpang thanks, I've added it. I think, necessity of license should be added somewhere in docs.